README.md

```diff
@@ -13,6 +13,16 @@ See the [documentation](https://appsody.dev/docs) for more details:
 * [Installation](https://appsody.dev/docs/getting-started/installation)
 * [CLI Reference](https://appsody.dev/docs/using-appsody/cli-commands)
 
+For information on building the Appsody CLI from source, see [Building from Source](build.md)
 
+## Contributing
```

Comment:
The stacks repo is missing the Contributing section

Reply:
The contributing section is within the `creating a stack` section. I can make this clearer by adding a separate section for it?